src/shared/components/ui/Calendar/CalendarHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, Heading } from "react-aria-components";
+import { tv } from "tailwind-variants";
+
+const calendarHeader = tv({
+  slots: {
+    root: "flex justify-between pb-3",
+    previous: "text-gray-400",
+    next: "text-gray-400",
+    heading: "font-semibold first-letter:uppercase",
+  },
+});
+
+const { root, previous, next, heading } = calendarHeader();
+
+export function CalendarHeader() {
+  return (
+    <header className={root()}>
+      <Button className={previous()} slot='previous'>
+        ◀
```

Comment:
Do we want to have the icon like this or perhaps adding an SVG component?

Reply:
I think that depends on whether we want to make a decision on icon styles within the boilerplate.
We could add something [like this](https://phosphoricons.com/) as a base and then if a project needs custom icons they can delete it and implement a different style.

Reply:
Perhaps add a prop to receive the left and right arrow button and add these icons as default, what do you think?